src/main/webapp/app/entities/course/course.component.html

```diff
@@ -55,7 +57,10 @@ <h4 id="course-page-heading" jhiTranslate="artemisApp.course.home.title">Courses
                 <tr *ngFor="let course of courses | sortBy: predicate:reverse; trackBy: trackId">
                     <ng-container *ngIf="!showOnlyActive || !course.endDate || (course.endDate | amIsAfter: today:'second')">
                         <td>
-                            <a [routerLink]="['/course', course.id, 'view']">{{ course.id }}</a>
+                            <div [ngStyle]="{ 'backgroundColor': course.color, 'width': '15px', 'height': '20px' }"> &nbsp;</div>
```

Comment:
Hi, I applied prettier locally and got diffs in line 60 and 63, perhaps that's whats breaking your build